openid-federation-1_0.xml

```diff
@@ -6610,10 +6619,10 @@ HTTP/1.1 302 Found
                   <t hangText="aud">
                     <vspace/>
                     REQUIRED.
-                    Its value MUST be the Entity Identifier of the RP. See
-                    <xref target="RFC7519"/> for the full specification. This
-                    claim is specific to Explicit Registration responses and is
-                    not a general Entity Statement claim.
+                    Its value MUST be the Entity Identifier of the RP.
+		    See <xref target="RFC7519"/> for the full specification.
+		    This claim is used in Explicit Registration responses
+		    but is not a general Entity Statement claim.
```

Comment:
To keep in mind that this issue relates to the "aud" in Entity Statement claims in response to authenticated callers (with `private_key_jwt`):

https://github.com/openid/federation/issues/98